packages/manager-react-components/src/components/notifications/notifications.spec.tsx

```diff
@@ -35,11 +35,27 @@ describe('notifications component', () => {
     container = await render(<Notifications />).container;
     expect(container.children.length).toBe(2);
   });
-  it('should clear notifications', async () => {
+  it('should not clear notifications created within the last 1000ms', async () => {
     let { container } = render(<Notifications />);
-    expect(container.children.length).not.toBe(0);
+    expect(container.children.length).toBe(2);
+
     render(<ClearNotifications />);
     container = render(<Notifications />).container;
-    expect(container.children.length).toBe(0);
+
+    expect(container.children.length).toBe(2);
+  });
+
+  it('should clear notifications older than 1000ms', async () => {
+    let { container } = render(<Notifications />);
+    expect(container.children.length).toBe(2);
+
+    await new Promise((resolve) => setTimeout(resolve, 2000));
+
+    render(<ClearNotifications />);
+    container = render(<Notifications />).container;
+
+    await waitFor(() => {
+      expect(container.children.length).toBe(0);
+    });
```

Comment:
I think it's better mock Date. 

- It's a unit test we don't test Data.now() method
- This allow to do the test faster

Reply:
enhanced the tests according to your suggestions